### PR TITLE
Disable codecov comments

### PR DESCRIPTION
**Pull request by MorrisJobke**
Was requested in https://github.com/nextcloud/server/pull/1903#issuecomment-256802739

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @LukasReschke @jancborchardt

**Comment by MorrisJobke** [comment]
cc @eppfel @ChristophWurst @te-online

**Comment by eppfel** [comment]
👍

**Comment by ChristophWurst** [comment]
👍 🙏 thanks

**Comment by LukasReschke** [comment]
Why? I do dislike that 👎

**Comment by LukasReschke** [comment]
Its an easy way to see which new files have received tests and which don't etc…

**Comment by LukasReschke** [comment]
And seriously, if you open a pull request you can VERY much live with the fact that you get a comment for new pushes etc… 

It's not like you open 300 of them a day…

And if somebody repushes on a PR you've already reviewed this means you should better take another look at it…

File: .codecov.yml [hunk 1]
```diff
@@ -6,6 +6,4 @@ coverage:
   round: down
   range: "70...100"
 
-comment:
-  layout: "header, diff, changes, sunburst, uncovered, tree"
```
Comment (LukasReschke) [inline .codecov.yml]: At least header should really stay… Removing this is just another easy way to ignore coverage. Awesome! 🚀

Comment (LukasReschke) [inline .codecov.yml]: (don't care about the other values)

**Comment by LukasReschke** [comment]
> Why? I do dislike the removal hugely 👎

And "Jan is annoyed by being notified once on each of his pull request" is _NOT_ a valid reason before that comes up.

**Comment by eppfel** [comment]
@LukasReschke I'm open to reconsidered, as I think there is a conflict of interest between developers focus on testing and others that are annoyed of the comments:grin:. But if you want to see changes, it should be the tree or diff view, not the header.
But why not improve the check / commit status. You could be more strict and then you can't ignore it.
Move discussion to an issue?

**Comment by rullzer** [comment]
I'm with @LukasReschke here. It is an easy overview.

**Comment by jancborchardt** [comment]
@LukasReschke @rullzer the reason for removing this is because it makes the discussion of a pull request hugely technical, and intimidating for any new contributors.

> most people here do simply ignore the results in the bar below.

If you ignore the results in the bar below, then pushing them up everyone’s faces is not going to change that. It will just result in being annoyed.

**Comment by jancborchardt** [comment]
It’s like having a bot posting to an IRC channel – misunderstanding the medium. A Github thread is a discussion between humans, not a space for bots to dump their results.

**Comment by rullzer** [comment]
Well if people changed code they should add/change tests. Having a clear overview of what was added etc is a lot easier then telling. Click on the status of codecov. Then see where you need to add coverage.

A github PR is the perfect place for that info. It is where we discuss if a new piece of code should go into the project. I'm sorry but a pull request is technical.

**Comment by LukasReschke** [comment]
This is just leading to having coverage ignored… Just had another PR today especially because the status simply got ignored. Jesus. Folks. This is proper software development and not a YOLO here.

Just because some of you are like "OMG I GET NOTIFIED ONCE" we can't just throw away tools ❗️

**Comment by LukasReschke** [comment]
> @LukasReschke @rullzer the reason for removing this is because it makes the discussion of a pull request hugely technical, and intimidating for any new contributors.

You gotta be kidding me, right? 

A PULL REQUEST _IS_ TECHNICAL. THIS *IS* ABOUT CODE.

Sorry to say it that clear. But this is bullshit.

**Comment by LukasReschke** [comment]
Revert in https://github.com/nextcloud/server/pull/1982

**Comment by LukasReschke** [comment]
> If you ignore the results in the bar below, then pushing them up everyone’s faces is not going to change that. It will just result in being annoyed.

The feedback in the result bar is _DIFFERENT_ from what is shown in the list. The feedback in the bar will also be green even if coverage effectively decreased. See https://github.com/nextcloud/server/pull/1347

**Comment by te-online** [comment]
> it makes the discussion of a pull request hugely technical

*Technical* as in not (instantly) *human readable*. 

I think there is nothing wrong **with the information itself**, but with the way how a (obviously) machine-generated output mixes with a conversation of (hopefully) real people. It doesn't feel right, because it is embedded in the conversation context (and not distinguishable from people's comments), but not really part of that conversation. 

One or two sentences, describing the current status in human readable language (and those could of course be machine-generated) and a maybe link for more details would be just enough. But I guess that not a configurable option, so for now I would support the revert as well.

**Pull request merged**